src/DotNetWorker/Http/SameSite.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    public enum SameSite
```

Comment:
Do we need to replicate every type like this? For example, we use `System.Net.HttpStatusCode` in responses.